### PR TITLE
adding dax action update-cluster

### PR DESCRIPTION
**Pull request by joshuaroot**
Re #2239 
- adding `update-cluster` action for dax

### CODE REVIEW

**Comment by kapilt** [comment]
while this is literal on the issue, its not geared towards fitting into how we modify sgs on the rest of the resources, and there for doesn't play nice with other patterns we have for sg filter and sg modify groups.

**Comment by joshuaroot** [comment]
Got it. Should I scrap this PR then?

**Comment by kapilt** [comment]
i think it probably provides value for the other modifyable values, but i'd like to keep the policies and enforcement on network controls uniform across network attached resources.

**Comment by joshuaroot** [comment]
gotcha. i'll migrate the SG aspect into it's own action following other implementations

File: c7n/resources/dynamodb.py [hunk 1]
```diff
@@ -670,3 +670,70 @@ def process(self, resources):
...
+        params.pop('type')
+        for r in resources:
+            params['ClusterName'] = r['ClusterName']
+            try:
+                client.update_cluster(**params)
+            except ClientError as e:
+                if e.response['Error']['Code'] in (
+                        'ClusterNotFoundFault',
+                        'InvalidClusterStateFault',
+                        'ParameterGroupNotFoundFault'):
```
Comment (kapilt) [inline c7n/resources/dynamodb.py]: parameter group not found should not be caught, its a persistent error that needs human attention.

**Pull request merged**